sorc/link_workflow.sh

```diff
@@ -131,6 +131,10 @@ fi
 #---------------------------------------
 #--add files from external repositories
 #---------------------------------------
+#--copy/link NoahMp table form ccpp-physics repository
+cd "${HOMEgfs}/parm/ufs/land" || exit 1
+  ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/FV3/ccpp/physics/physics/noahmptable.tbl" .
```

Comment:
Please remove the `noahmptable.tbl` from the global-workflow repository and link from the UFS-weather-model.